.ci/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,146 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+import org.kie.jenkins.MavenCommand
+
+changeAuthor = env.ghprbAuthorRepoGitUrl ? util.getGroup(env.ghprbAuthorRepoGitUrl) : (env.ghprbPullAuthorLogin ?: CHANGE_AUTHOR)
+changeBranch = env.ghprbSourceBranch ?: CHANGE_BRANCH
+changeTarget = env.ghprbTargetBranch ?: CHANGE_TARGET
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && kie-mem16g'
```

Comment:
do we need 16G for kie-editors-java ?

Reply:
I thought yes, however lets try 8G

https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/jomarko/job/kogito-editors-java-main-pr-sonar-test/13/